.circleci/config.yml

```diff
@@ -7,89 +7,51 @@
 # GCLOUD_SERVICE_KEY - key for gsutil rsync bootstrap and jobs with gcs
 # GOOGLE_PROJECT_ID - airflow-dataproc
 # GOOGLE_COMPUTE_ZONE - us-west1-b
-#
 
-version: 2
-jobs:
-  build:
-    docker:
-      - image: docker:18.02.0-ce
-    working_directory: ~/mozilla/telemetry-airflow
-    steps:
-      - checkout
-      - setup_remote_docker
-      - run: |
-          printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' "$CIRCLE_SHA1" "$CIRCLE_TAG" "$CIRCLE_PROJECT_USERNAME" "$CIRCLE_PROJECT_REPONAME" "$CIRCLE_BUILD_URL" > version.json
```

Comment:
this stuff is needed still. This is how we build images for dockerhub and jenkins jobs do a verification step to ensure the file exists inside the container.

Reply:
version.json

Reply:
got it, added back!